### PR TITLE
Run cluster resource as a deployment not cron

### PR DESCRIPTION
**Pull request by katrogan**
Signed-off-by: Katrina Rogan <katroganGH@gmail.com>

The run command for cluster resource controller runs it periodically according to the configured refresh interval. Running the entire process as a pod deployment instead of a kubernetes cron greatly reduces spam from frequent and uniquely identified job invocations. This impacts label cardinality _which is useful for saving costs with monitoring/observation SaaS providers_.

### CODE REVIEW

File: charts/flyte-core/values-gcp.yaml [hunk 1]
```diff
@@ -313,10 +313,17 @@ configmap:
...
+  service_account_name: flyteadmin
+  # -- Annotations for ClusterResource pods
+  podAnnotations: {}
   config:
     cluster_resources:
+      # -- How frequently to run the sync process
+      refreshInterval: 5m
+      templatePath: "/etc/flyte/clusterresource/templates"
+      # -- Starts the cluster resource manager in standalone mode with requisite auth credentials to call flyteadmin service endpoints
+      standaloneDeployment: false
```
Comment (EngHabu) [inline charts/flyte-core/values-gcp.yaml:326]: Do we need these here if they are already in the values.yaml file?

Comment (katrogan) [reply charts/flyte-core/values-gcp.yaml]: oh right, reverted!

**Comment by katrogan** [comment]
PTAL @EngHabu

**Review by evalsocket — DISMISSED** [review]

**Comment by katrogan** [comment]
PTAL @EngHabu @evalsocket

**Review by evalsocket — APPROVED** [review]

**Pull request merged**